ui/packages/consul-ui/app/components/consul/auth-method/binding-view/index.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="consul-auth-method-binding-view">
+{{#if (gt @items.length 0)}}
+  <p>Binding rules allow an operator to express a systematic way of automatically linking roles and service identities to newly created tokens without operator intervention.<br>
+Successful authentication with an auth method returns a set of trusted identity attributes corresponding to the authenticated identity. Those attributes are matched against all configured binding rules for that auth method to determine what privileges to grant the the Consul ACL token it will ultimately create.</p>
+  <hr />
+  {{#each @items as |item|}}
+  <h2>{{item.BindName}}</h2>
+  <dl>
+  {{#each (array "BindType" "Selector" "Description") as |value|}}
+    {{#if (get item value)}}
+    <dt>{{t (concat "models.binding-rule." value)}}</dt>
+    <dd>{{get item value}}</dd>
+    {{/if}}
+  {{/each}}
+  </dl>
+  <hr />
+  {{/each}}
+{{else}}
+  <EmptyState>
+    <BlockSlot @name="header">
+      <h2>No binding rules</h2>
+    </BlockSlot>
+    <BlockSlot @name="body">
+      <p>Binding rules allow an operator to express a systematic way of automatically linking roles and service identities to newly created tokens without operator intervention.</p>
+    </BlockSlot>
+    <BlockSlot @name="actions">
+      <li class="docs-link">
+        <a href="{{env 'CONSUL_DOCS_API_URL'}}/acl/binding-rules" rel="noopener noreferrer" target="_blank">Read the documentation</a>
+      </li>
```

Comment:
I 'think' this link needs to go to the k8s specific docs if the auth method is a k8s one. Have a look in the designs mocks, theres a note there about it.

Reply:
I saw that. There are no specific k8 binding rules docs.